### PR TITLE
Project week 10 

### PR DESCRIPTION
**Pull request by Pauan8**
Survey

### CODE REVIEW

File: code/src/components/ToggleQuestions.js [hunk 1]
```diff
@@ -0,0 +1,191 @@
...
+
+          {questionNumber === "Question5" && (
+            <section className="question" aria-labelledby="Question5">
+              <h2 className="question-heading" id="Question5">
+                Interesting, tell me more!
+              </h2>
+              <ToggleSubQuestions
+                futureChoice={futureChoice}
+                setMoreFutureChoices={setMoreFutureChoices}
+                moreFutureChoices={moreFutureChoices}
```
Comment (Aepera) [inline code/src/components/ToggleQuestions.js:150]: It seems like I could skip choosing a checkbox here on question 5 and could still click on Submit.

**Review by Aepera — APPROVED** [review]
Very well executed project! 🎉The application contains several different form input types as well as some nice css styling tricks elevating the experience of the website. I like the structure with each question on a new page.

The application is working well, the only details I can feedback on are these things not working on iPhone 6: the animation on start page (Aliens/Hoverboard...) and the dropdown menu.

File: code/src/App.js [hunk 2]
```diff
@@ -1,9 +1,34 @@
...
+      <header className="header" aria-labelledby="header">
+        <Header title="The FUTURE" />
+      </header>
+
+      {questionNumber === "" && (
+        <main>
+        <div className="start-page" aria-labelledby="start-heading">
+          <h2 className="question-heading" id="start-heading">Let's talk about the future!</h2>
+        </div>
+        </main>
```
Comment (KarolCislowski) [inline code/src/App.js:21]: You could create something like `Welcome.js` to make your app.js only decides which components will be rendered.

**Review by KarolCislowski — APPROVED** [review]
`Your own words about the future: year 2050 Vladimir Putin chosen as President of Russia. /Genius` - Better not... 😨 

Great job 👍🏻

**Pull request closed without merging**